Gemfile

```diff
@@ -58,3 +58,4 @@ gem 'net-pop'
 gem 'net-smtp'
 
 gem 'carrierwave'
+gem 'i18n_generators'
```

Comment:
[nits] `i18n_generators` の使い方として、開発時に `$ rails g i18n_translation` を実行するためのものというところがあるので、`group :development do 〜 end` の中に含めるのが良いですね :eyes: 

https://github.com/amatsuda/i18n_generators#installation

> Add 'i18n_generators' to your Gemfile, then bundle. Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.

Reply:
レビューありがとうございます！

> [nits] i18n_generators の使い方として、開発時に $ rails g i18n_translation を実行するためのものというところがあるので、group :development do 〜 end の中に含めるのが良いですね 👀

なるほどです！段階によってインストールしたいgemが違うので分けているのですね！

お一つ質問させていただきたいのですが、`bundle install`のオプションで`--without development test`を指定すると本番環境で、テストと開発環境のグループを除いたgemをインストールすることができると思うのですが、オプションを指定していない場合はGemfileに記載した全てのgemがインストールされるのでしょうか。（実際に`bundle install`をしてみたのですが全てのgemがインストールされていました。）

Reply:
こちらのコミットで修正いたしました、ご確認よろしくお願いします。

[44bb4d4](https://github.com/OdenTakashi/fjord-books_app/pull/1/commits/44bb4d46764e24fed9739b4968621f53fa035e79)

Reply:
そうですね、bundlerの機能としては、オプションで明示的に除外設定がなければ、全てのgemがインストールされることになると思います。
Railsに絞っていうと、 `config/application.rb` の中で次のように書かれています。

```ruby
Bundler.require(*Rails.groups)
```

これによって、default(何のグループにも属していないgem)と、指定の開発環境( 主に development  OR  test  OR  production)のgemが入るようです。

なので、 `:hoge` とか何も関係のないグループがRails上のGemfileに書いてあっても、それは無視されるかもしれないと思いました〜 👀  (すみません、動作確認まではできておりませんが...! 🙇  )

詳しくは公式ページを参照していただくのが良さそうです 👍 
[Bundler: How to manage groups of gems](https://bundler.io/guides/groups.html)

Reply:
@betachelsea さん
ご回答していただきありがとうございます！

> Railsに絞っていうと、 config/application.rb の中で次のように書かれています。
~~~
Bundler.require(*Rails.groups)
~~~
> これによって、default(何のグループにも属していないgem)と、指定の開発環境( 主に development OR test OR production )のgemが入るようです。

このような設定がされていたのですね！
オプション以外でもインストールするgemを選ぶことができるのですね！

参考ページを教えていただきありがとうございます！